examples/article_info_using_dictionary.py

```diff
@@ -22,3 +22,9 @@
 print(first_step.title)
 print(first_step.description)
 print(first_step.picture)
+
+print(info['co_authors'])
```

Comment:
Shouldn't we keep all print statements of article data members together? After line 17.

Reply:
Okay, I will do that.